mobilesdk/zbox/storage.go

```diff
@@ -334,20 +335,23 @@ func MultiUpload(allocationID string, workdir string, jsonMultiUploadOptions str
 	thumbnailPaths := make([]string, totalUploads)
 	encrypts := make([]bool, totalUploads)
 	chunkNumbers := make([]int, totalUploads)
+	isUpdates := make([]bool, totalUploads)
 	for idx, option := range options {
 		filePaths[idx] = option.FilePath
 		fileNames[idx] = option.FileName
 		thumbnailPaths[idx] = option.ThumbnailPath
 		remotePaths[idx] = option.RemotePath
 		chunkNumbers[idx] = option.ChunkNumber
+		isUpdates[idx] = false
 		encrypts[idx] = option.Encrypt
+    isUpdates[idx] = false
```

Comment:
This `isUpdates[idx] = false` is set twice, see line 345 above

Reply:
fixed